src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -851,7 +852,8 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
                aad_admin_group_object_ids=None,
                disable_sgxquotehelper=False,
                assign_identity=None,
-               no_wait=False):
+               no_wait=False,
+               osm_mesh_name=None):
```

Comment:
Nit: move this up to be with the other addons or at least above no_wait

Reply:
Done.